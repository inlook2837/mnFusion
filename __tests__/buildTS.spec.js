jest.setTimeout(600000)

const runTests = require('./build.helper.js').runTests

test('electron:build-ts', async () => {
  await runTests(true)
})
