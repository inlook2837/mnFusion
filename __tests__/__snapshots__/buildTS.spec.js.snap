// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`build:electron-ts 1`] = `
"<div id=\\"app\\"><img src=\\"./img/logo.82b9c7a5.png\\"><div data-v-27fc7cba=\\"\\" class=\\"hello\\"><h1 data-v-27fc7cba=\\"\\">Welcome to Your Vue.js + TypeScript App</h1><p data-v-27fc7cba=\\"\\">
    For guide and recipes on how to configure / customize this project,<br data-v-27fc7cba=\\"\\">
    check out the
    <a data-v-27fc7cba=\\"\\" href=\\"https://cli.vuejs.org\\" target=\\"_blank\\">vue-cli documentation</a>.
  </p><h3 data-v-27fc7cba=\\"\\">Installed CLI Plugins</h3><ul data-v-27fc7cba=\\"\\"><li data-v-27fc7cba=\\"\\"><a data-v-27fc7cba=\\"\\" href=\\"https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-babel\\" target=\\"_blank\\">babel</a></li><li data-v-27fc7cba=\\"\\"><a data-v-27fc7cba=\\"\\" href=\\"https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-eslint\\" target=\\"_blank\\">eslint</a></li><li data-v-27fc7cba=\\"\\"><a data-v-27fc7cba=\\"\\" href=\\"https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-typescript\\" target=\\"_blank\\">typescript</a></li><li data-v-27fc7cba=\\"\\"><a data-v-27fc7cba=\\"\\" href=\\"https://nklayman.github.io/vue-cli-plugin-electron-builder/guide/testingAndDebugging.html#debugging\\" target=\\"_blank\\">electron-builder</a></li></ul><h3 data-v-27fc7cba=\\"\\">Essential Links</h3><ul data-v-27fc7cba=\\"\\"><li data-v-27fc7cba=\\"\\"><a data-v-27fc7cba=\\"\\" href=\\"https://vuejs.org\\" target=\\"_blank\\">Core Docs</a></li><li data-v-27fc7cba=\\"\\"><a data-v-27fc7cba=\\"\\" href=\\"https://forum.vuejs.org\\" target=\\"_blank\\">Forum</a></li><li data-v-27fc7cba=\\"\\"><a data-v-27fc7cba=\\"\\" href=\\"https://chat.vuejs.org\\" target=\\"_blank\\">Community Chat</a></li><li data-v-27fc7cba=\\"\\"><a data-v-27fc7cba=\\"\\" href=\\"https://twitter.com/vuejs\\" target=\\"_blank\\">Twitter</a></li></ul><h3 data-v-27fc7cba=\\"\\">Ecosystem</h3><ul data-v-27fc7cba=\\"\\"><li data-v-27fc7cba=\\"\\"><a data-v-27fc7cba=\\"\\" href=\\"https://router.vuejs.org\\" target=\\"_blank\\">vue-router</a></li><li data-v-27fc7cba=\\"\\"><a data-v-27fc7cba=\\"\\" href=\\"https://vuex.vuejs.org\\" target=\\"_blank\\">vuex</a></li><li data-v-27fc7cba=\\"\\"><a data-v-27fc7cba=\\"\\" href=\\"https://github.com/vuejs/vue-devtools#vue-devtools\\" target=\\"_blank\\">vue-devtools</a></li><li data-v-27fc7cba=\\"\\"><a data-v-27fc7cba=\\"\\" href=\\"https://vue-loader.vuejs.org\\" target=\\"_blank\\">vue-loader</a></li><li data-v-27fc7cba=\\"\\"><a data-v-27fc7cba=\\"\\" href=\\"https://github.com/vuejs/awesome-vue\\" target=\\"_blank\\">awesome-vue</a></li></ul></div></div>"
`;
